### ISSUE TITLE
Enhancement: select boxes for coordinate reference systems supported by GeoTools

### ISSUE THREAD

**Issue by jindrichmynarz** [linkedpipes/etl#287]
Similarly to #281, it would be useful if the GeoTools component's inputs for "Default coordinates type" and "Type of output coordinates" were select boxes (or autocomplete-enabled combo-boxes) that list only the coordinate reference systems (CRSs) supported by GeoTools. I suspect there is a way to get a list of CRSs supported by GeoTools automatically, so this could feed the select box.

**Comment by skodapetr** [comment]
@jindrichmynarz Do you have any suggestions how to "get a list of CRSs supported by GeoTools automatically" ?

**Comment by jindrichmynarz** [comment]
One way to do that is this:

```java
import org.geotools.referencing.CRS;

CRS.getSupportedCodes("EPSG");
```

**Comment by jakubklimek** [comment]
This list is quite long (5K entries), I am not sure whether it makes sense for it to be part of the dialog. Maybe we should attach it to the component documentation.

**Comment by jindrichmynarz** [comment]
Yes, it would make more sense if only provided as a dynamic autocomplete. However, I suspect that would require a more thorough change in the architecture of LP-ETL, since its API would have to expose an endpoint for autocompletion.

**Comment by skodapetr** [comment]
@jindrichmynarz @jakubklimek So it should be removed from the dialog and moved to documentation?

**Comment by jindrichmynarz** [comment]
While the autocomplete backed by client-side data looks nice, it is quite slow. Perhaps more specific search function can be specified that would match only the numeric codes of the supported CRSs, left-anchored.

**Comment by jindrichmynarz** [comment]
I think the ideal situation is to have this as an autocomplete, but backed by data dynamically retrieved from the server.

**Comment by jakubklimek** [comment]
OK. This is then actually a special case of https://github.com/linkedpipes/etl/issues/87. As this would be a bigger thing to implement, I move this to not planned.
